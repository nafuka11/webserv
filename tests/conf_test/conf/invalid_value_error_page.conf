server {
    location /error_page {
        error_page 100 100.html;
    }
}