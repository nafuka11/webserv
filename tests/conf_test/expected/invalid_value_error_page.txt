webserv: invalid value in "error_page" directive in ./conf/invalid_value_error_page.conf:3
